### PR TITLE
 remove clearing cache for integration tests

### LINKED ISSUE #31074: [Issue] [WIP] remove clearing cache for integration tests
This issue is automatically created based on existing pull request: magento/magento2#31053: [WIP] remove clearing cache for integration tests

---------
Removed `CacheCleaner::cleanAll();` for integration tests

### PR DESCRIPTION
**Pull request by Usik2203**
Removed `CacheCleaner::cleanAll();` for integration tests 

### Resolved issues:
1. [x] resolves magento/magento2#31074: [WIP] remove clearing cache for integration tests

### CODE REVIEW

**Comment by Usik2203** [comment]
@magento run all tests

**Comment by Usik2203** [comment]
@magento run Integration Tests

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]
Hi @Usik2203,
Please check failing tests and fix them

**Comment by gabrieldagama** [comment]
@magento create issue

File: dev/tests/api-functional/testsuite/Magento/GraphQl/CatalogUrlRewrite/UrlResolverTest.php [hunk 1]
```diff
@@ -224,8 +224,6 @@ public function testRedirectsAndCustomInput()
         $urlRewriteModel->setId($urlRewriteModel->getId());
         $urlRewriteModel->save();
```
Comment (Usik2203) [inline dev/tests/api-functional/testsuite/Magento/GraphQl/CatalogUrlRewrite/UrlResolverTest.php]:
```suggestion
ObjectManager::getInstance()->get(\Magento\TestFramework\Helper\CacheCleaner::class)->clean(['eav']);
```

File: dev/tests/api-functional/testsuite/Magento/GraphQl/CatalogUrlRewrite/UrlResolverTest.php [hunk 2]
```diff
@@ -223,9 +223,7 @@ public function testRedirectsAndCustomInput()
         $urlRewriteModel->setRedirectType('301');
         $urlRewriteModel->setId($urlRewriteModel->getId());
         $urlRewriteModel->save();
-
-        ObjectManager::getInstance()->get(\Magento\TestFramework\Helper\CacheCleaner::class)->cleanAll();
-
+ObjectManager::getInstance()->get(\Magento\TestFramework\Helper\CacheCleaner::class)->clean(['eav']);
```
Comment (Usik2203) [inline dev/tests/api-functional/testsuite/Magento/GraphQl/CatalogUrlRewrite/UrlResolverTest.php]:
```suggestion
        ObjectManager::getInstance()->get(\Magento\TestFramework\Helper\CacheCleaner::class)->clean(['eav']);
```

**Comment by Usik2203** [comment]
@magento run WebAPI Tests

**Comment by engcom-Hotel** [comment]
@magento run all tests

**Comment by engcom-Hotel** [comment]
@magento run all tests

**Comment by engcom-Hotel** [comment]
@magento run all tests

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests B2B

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests B2B

**Comment by engcom-Hotel** [comment]
Hello @Usik2203 ,
Please check Integration and API tests, could be related to PR changes.

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests B2B

**Comment by engcom-Hotel** [comment]
@magento run all tests

**Comment by engcom-Charlie** [comment]
@Usik2203 I'll try to continue with your PR.

**Comment by engcom-Charlie** [comment]
@magento run all tests

File: dev/tests/api-functional/testsuite/Magento/GraphQl/CatalogUrlRewrite/UrlResolverTest.php [hunk 3]
```diff
@@ -219,12 +219,10 @@ public function testRedirectsAndCustomInput()
         $urlRewriteModel->setRedirectType('301');
         $urlRewriteModel->setId($urlRewriteModel->getId());
         $urlRewriteModel->save();
-
-        ObjectManager::getInstance()->get(\Magento\TestFramework\Helper\CacheCleaner::class)->cleanAll();
         //modifying query by adding spaces to avoid getting cached values.
         $this->queryUrlAndAssertResponse(
             (int) $product->getEntityId(),
-            $customUrl,
+            $customUrl . ' ',
```
Comment (ihor-sviziev) [inline dev/tests/api-functional/testsuite/Magento/GraphQl/CatalogUrlRewrite/UrlResolverTest.php:225]: Could you explain why this change was added and why it wasn't needed earlier?

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]
Maybe it's a good idea to deprecate using the method `CacheCleaner::cleanAll()`?

File: dev/tests/api-functional/testsuite/Magento/GraphQl/CatalogUrlRewrite/UrlResolverTest.php (hunk 3 above)
Comment (engcom-Charlie) [reply dev/tests/api-functional/testsuite/Magento/GraphQl/CatalogUrlRewrite/UrlResolverTest.php:225]: It was added as described in comment https://github.com/magento/magento2/pull/31053/files/9ec34d8ca0a27ba9e34d8ad4803b6217175605d9#diff-e996da3d86e3da3160dd34c221e07fd9c45874f3ea3795b574e7e09ed004fad3R222
Since earlier we use `CacheCleaner::cleanAll()` and we got not cached response.

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B

**Review by ihor-sviziev — APPROVED** [review]
✔ Approved.

Failing tests looks not related to changes form this PR.

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-8529](https://jira.corp.magento.com/browse/ENGCOM-8529) has been created to process this Pull Request
:eight_spoked_asterisk: @ihor-sviziev, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
QA not applicable
